ludwig/schema/utils.py

```diff
@@ -1034,3 +1034,44 @@ def is_primitive(value):
         },
         **default_kwarg,
     )
+
+
+@DeveloperAPI
```

Comment:
I don't think either of these classes are necessary.

Reply:
commented out, TODO to clean u[p before merge